### DISCUSSION TITLE
Newbie: Be Gentle

### DISCUSSION THREAD

**Discussion by kenryanjr** [templaza/astroid-framework#140] [General]
I need to add a script src link to the popper.js for astroid zero but cannot figure out in what file all of the js files get called.

**Comment by Chacapamac** [comment]
I don’t think you can make an override. You will have to make a hack.

media/astroid/assets/vendor/bootstrap/js/popper.min.js

**Comment by kenryanjr** [comment]
Is there somewhere I can add the file so in the compiler to add it automatically? Funny thing is, popper is included in the framework.

On Jun 14, 2022, at 7:41 AM, Chacapamac ***@***.***> wrote:


I don’t think you can make an override. You will have to make a hack.

media/astroid/assets/vendor/bootstrap/js/popper.min.js

**Comment by Chacapamac** [comment]
I’m not sure if it help but this is the places where you can add Custom JS files in Astroid.

**• In the Template Styles **
_Use **Default** to add js to all website or a **Custom style** to impact a specific section of the website_
![template-js](https://user-images.githubusercontent.com/1907276/175774547-35ce566a-a755-44e2-bab3-be5a44b0807b.jpg)

**• in the Joomla menu item (will be added only to that menu)**
![menu-js](https://user-images.githubusercontent.com/1907276/175774667-d473e87e-c63b-4955-bd64-7bfc602a8f30.jpg)

**• Directly in template js folder**
_Will be loaded on all website_
![direct-js](https://user-images.githubusercontent.com/1907276/175774790-a616000e-fd20-49a6-803e-a150e3e067d3.jpg)
